DESCRIPTION

```diff
@@ -16,12 +16,19 @@ Imports:
     broomstick,
     car,
     cowplot,
+<<<<<<< HEAD
```

Comment:
For future reference, please note the error messages in your console when you try to merge! These tags indicate areas where your branch conflicted with the main repository.

I'm fixing this (and getting rid of the extra directory you checked in), but note for future reference.